pkg/generate/app/componentref.go

```diff
@@ -42,20 +44,22 @@ func componentWithSource(s string) (component, repo string, builder bool, err er
 	} else {
 		component = s
 	}
-	// TODO: component must be of the form compatible with a pull spec *or* <namespace>/<name>
+	// component must be of the form compatible with a pull spec *or* <namespace>/<name>
```

Comment:
@smarterclayton removed this TODO since `ParseDockerImageReference` already checks for both cases, [ain't it](https://github.com/openshift/origin/blob/e6d024bc78fc682c081aba1a461bb57bc8577f77/pkg/image/api/helper.go#L44)?